CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 0.29.0
+
+- Please check
+  [COMPATIBILITY.md](https://github.com/packit/ogr/blob/main/COMPATIBILITY.md)
+  to see which methods are implemented for particular services.
```

Comment:
since many changes happened to this document, I'd like to highlight it exists :)